test/test.tf

```diff
@@ -5,6 +5,80 @@ provider "newrelic" {
   region        = var.newrelic_region
 }
 
+provider "pagerduty" {
```

Comment:
Can we move all this into a pagerduty.tf file and leave here only the module resource?